config/initializers/chartkick.rb

```diff
@@ -1,185 +1,186 @@
+        colors: ["#A9467A","#CF6456","#ECF8D7","#A5D7B2",
+                 "#D58CB2", "#FFB2A8", "#82C594", "#D2EF9D",
+                 "#0088CC"],
        chart: {
-      backgroundColor: "transparent",
+            backgroundColor: "transparent",
       style: {
-         fontFamily: "'Unica One', sans-serif"
+         fontFamily: "Unica One, sans-serif"
       },
-      plotBorderColor: '#606063'
+      plotBorderColor: "#606063"
    },
    title: {
       style: {
-         color: '#0088CC',
-         textTransform: 'uppercase',
-         fontSize: '20px'
+         color: "#0088CC",
+         textTransform: "uppercase",
+         fontSize: "20px"
       }
    },
    subtitle: {
       style: {
-         color: '#0088CC',
-         textTransform: 'uppercase'
+               color: "#0088CC",
+               textTransform: "uppercase"
       }
    },
  xAxis: {
-      gridLineColor: '#707073',
+      gridLineColor: "#707073",
       labels: {
          style: {
-            color: '#E0E0E3'
+            color: "#E0E0E3"
          }
       },
-      lineColor: '#707073',
-      minorGridLineColor: '#505053',
-      tickColor: '#707073',
+      lineColor: "#707073",
+      minorGridLineColor: "#505053",
+      tickColor: "#707073",
       title: {
          style: {
-            color: '#A0A0A3'
+            color: "#A0A0A3"
 
          }
       }
    },
    yAxis: {
-      gridLineColor: '#707073',
+      gridLineColor: "#707073",
       labels: {
          style: {
-            color: '#E0E0E3'
+            color: "#E0E0E3"
          }
       },
-      lineColor: '#707073',
-      minorGridLineColor: '#505053',
-      tickColor: '#707073',
+      lineColor: "#707073",
+      minorGridLineColor: "#505053",
+      tickColor: "#707073",
       tickWidth: 1,
       title: {
          style: {
-            color: '#A0A0A3'
+            color: "#A0A0A3"
          }
       }
    },
    tooltip: {
-      backgroundColor: 'rgba(0, 0, 0, 0.85)',
+      backgroundColor: "rgba(0, 0, 0, 0.85)",
       style: {
-         color: '#F0F0F0'
+         color: "#F0F0F0"
       }
    },
    plotOptions: {
       series: {
          dataLabels: {
-            color: '#0088CC'
+            color: "#000"
          },
          marker: {
-            lineColor: '#333'
+            lineColor: "#333"
          }
       },
       boxplot: {
-         fillColor: '#505053'
+         fillColor: "#505053"
       },
       candlestick: {
-         lineColor: 'white'
+         lineColor: "white"
       },
       errorbar: {
-         color: 'white'
+         color: "white"
       }
    },
    legend: {
       itemStyle: {
-         color: '#E0E0E3'
+         color: "#E0E0E3"
       },
       itemHoverStyle: {
-         color: '#FFF'
+         color: "#FFF"
       },
       itemHiddenStyle: {
-         color: '#606063'
+         color: "#606063"
       }
    },
    credits: {
       style: {
-         color: '#666'
+         color: "#666"
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.